python/pyspark/sql/functions.py

```diff
@@ -67,9 +67,16 @@ def _():
     _.__doc__ = 'Window function: ' + doc
     return _
 
+_lit_doc = """
+    Creates a :class:`Column` of literal value. Supports basic types like :class:`IntegerType`,
+    :class:`FloatType`, :class:`BooleanType`, and :class:`StringType`
```

Comment:
I would like to keep this identical with the one in `functions.scala` to reduce overhead when someone sweeps the same documentation changes across APIs in other languages.

If the additional informations are Python-specific, let's add it in `::note`.